### PR TITLE
refactor: Organise the Scrabble exercise into "background" and "tasks"

### PR DESCRIPTION
**Pull request by akash1810**
## What does this change?

This change aims to make the Scrabble exercise easier to follow.

Currently, the background information/facts are interleaved with the tasks.

This change creates two sections:
  1. "background" to state the letter scores and distribution in tabular form
  1. "tasks" to list the tasks in a true markdown list. Hopefully, this makes the order of tasks clearer.

This brings the Scrabble exercise in line with other exercises such as 21s and Game of Life.

Also adds a new header image.

See it [here](https://github.com/guardian/coding-exercises/blob/aa-scrabble/scrabble/README.md). The screenshot below also shows a before and after.

## How can we measure success?

The Scrabble exercise is easier to follow.

## Images

image

### CODE REVIEW

**Review by jonathonherbert — APPROVED** [review]
Brilliant. Was going to suggest changing 'distribution' to 'no. of tiles', but the tile suffix on each number makes that clear. I've run into clarity problems a few times in questions 1,2 & 3 and not thought to change this – thanks so much!

**Comment by akash1810** [comment]
> Was going to suggest changing 'distribution' to 'no. of tiles'

I started with this, but the two tables looked too similar and thought it might be a little confusing as a result. One might accidentally lookup scores in the distribution table when performing the exercise IRL and vice-versa.

**Comment by twrichards** [comment]
Great improvements. How do the tables look when copied and pasted into IDE? as lots of candidates copy/paste them into comments above their code when making their data structures.

**Comment by akash1810** [comment]
> How do the tables look when copied and pasted into IDE?

Still usable, though the alignment of `E` in distribution is questionable!

image

**Comment by rhystmills** [comment]
Looks great!

File: scrabble/README.md [hunk 1]
```diff
@@ -1,41 +1,50 @@
...
+
+## Tasks
+1. Calculate the score for a word. The score is the sum of the points for the letters that make up a word.
+   For example: GUARDIAN = 2 + 1 + 1 + 1 + 2 + 1 + 1 + 1 = 10.
+1. Assign seven tiles chosen randomly from the English alphabet to a player's rack.
+1. In the real game, tiles are taken at random from a 'bag' containing a fixed number of each tile.
+   Assign seven tiles to a rack using a bag containing the above distribution.
+1. Find a valid word formed from the seven tiles. A list of valid words can be found in [`dictionary.txt`](./dictionary.txt).
+1. Find the longest valid word that can be formed from the seven tiles.
+1. Find the highest scoring word that can be formed.
```
[the remark this answers is not in the document]
Comment (akash1810) [reply scrabble/README.md:48]: Agreed, this is a good clarification to make. Want to raise a PR?

**Pull request merged**